vaccine_feed_ingest/runners/az/arcgis/normalize.py

```diff
@@ -172,7 +179,7 @@ def _normalize_hours(
         return [
             schema.OpenHour(
                 day=day,
-                open=_normalize_time(open_time.strip().upper()),
+                opens=_normalize_time(open_time.strip().upper()),
```

Comment:
It looks like this logic drops a lot of hours simply because they have `AM` or `PM`. 
![image](https://user-images.githubusercontent.com/3347571/116801360-16535400-aabe-11eb-8d6a-e6c7d74d1bc8.png)

This is okay to fix in a followup because it's not a new issue, but... could you take a look at this (or open an issue for it)?

Reply:
thanks for the reminder! i caught this during my testing and then forgot to open an issue :(